README.md

```diff
@@ -86,8 +86,9 @@ kubectl label namespace appmesh-demo appmesh.k8s.aws/sidecarInjectorWebhook=enab
 
 ### Default behavior and how to override
 
-Sidecars will be injected to all new pods in the namespace that has enabled sidecar injector webhook. To disable injecting the sidecar
+By default, sidecars will be injected to all new pods in the namespace that has enabled sidecar injector webhook. To disable injecting the sidecar
```

Comment:
I think this would be slightly more clear if it was something more like: 

> For namespaces with sidecar injection enabled, pods will be injected if the `appmesh.k8s.aws/sidecarInjectorWebhook` annotation is `enabled` and will not be injected if it is `disabled`.  For pods with no annotation, they will be injected if `-inject-default=true` (the default for this flag) and will not be injected if `-inject-default=false`.